src/components/RespondToPetitionForm/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { reduxForm } from 'redux-form';
+import { respondToPetition } from 'actions/RespondActions';
+import petitionResponseValidator from 'form/petitionResponseValidator';
+import Fieldset from 'components/Fieldset';
+import FormField from 'components/FormField';
+import Button from 'components/Button';
+import settings from 'settings';
+import FIELDS from './fields';
+
+const RespondToPetitionForm = ({ petition, fields, handleSubmit, submitting }) => {
+  return (
+    <form onSubmit={handleSubmit(respondToPetition)}>
+      <Fieldset>
+        {FIELDS.map(field => (
+          <FormField
+            key={field.name}
+            config={field}
+            helper={fields[field.name]}
```

Comment:
@mattberridge, @Tom-Bonnike, @eschaefer

This won't work with nested form fields like `answer.text` or `answer.name`. I managed to solve this using `helper={eval('fields.' + field.name)}` instead but this feels a bit hacky. Is there a better way to do this?